CHANGELOG.md

```diff
@@ -0,0 +1,73 @@
+Changelog
+=========
+
+Unreleased
+----------
+### Added
```

Comment:
Should we add a note that we've migrated repos?

Reply:
eh, that shouldn't affect our consumers

Reply:
no but component history 🤷‍♀